plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryInstanceCleaner.java

```diff
@@ -113,7 +114,7 @@ public void cleanUpTables(String schemaName)
     private void dropTablesCreatedBeforeYesterday(String schemaName)
     {
         LOG.info("Identifying tables to drop...");
-        // Drop all tables created before yesterday (in UTC)
+        // Drop all tables created more than 24 hours ago
```

Comment:
Does this require changing the formula below?

eg something like

```
date_diff(current_timestamp(), date(creation_time), HOUR) > 24

Reply:
DAY is the smallest unit for date_diff. Also current formula is "correct" since only tables older than 24 hours will get dropped (but not all of them).

I'd let it be (maybe adjust the comment a bit)?

Reply:
Nevermind, changed to use `datetime_diff` which supports this.